### PR TITLE
docs: translate content/docs/thinking-in-react.md and other related

### PR DESCRIPTION
**Pull request by warrenca**

### CODE REVIEW

File: content/docs/thinking-in-react.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+title: Paano mag-isip sa React
 permalink: docs/thinking-in-react.html
 redirect_from:
   - 'blog/2013/11/05/thinking-in-react.html'
   - 'docs/thinking-in-react-zh-CN.html'
 prev: composition-vs-inheritance.html
 ---
 
-React is, in our opinion, the premier way to build big, fast Web apps with JavaScript. It has scaled very well for us at Facebook and Instagram.
+Ang React ay, sa aming opinyon, nangunguna sa pagbuo ng malaki at mabilis na Web apps gamit ang Javascript. Ito ay mabisa naming nagamit sa Facebook at Instagram.
```
Comment (elrmndo) [inline content/docs/thinking-in-react.md:11]:
```suggestion
Ang React, sa aming opinyon, ay nangunguna pagdating sa pagbuo ng malaki at mabilis na Web apps gamit ang Javascript na mabisa naming nagamit sa Facebook at Instagram.
```

**Comment by warrenca** [comment]
Hi @elrmndo, I have updated this PR based on your comments, pls have a look again. thanks

File: content/docs/thinking-in-react.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
   - 'blog/2013/11/05/thinking-in-react.html'
   - 'docs/thinking-in-react-zh-CN.html'
 prev: composition-vs-inheritance.html
 ---
 
-React is, in our opinion, the premier way to build big, fast Web apps with JavaScript. It has scaled very well for us at Facebook and Instagram.
+Ang React, sa aming opinyon, ay nangunguna pagdating sa pagbuo ng malaki at mabilis na Web apps gamit ang Javascript na mabisa naming nagamit sa Facebook at Instagram.
 
-One of the many great parts of React is how it makes you think about apps as you build them. In this document, we'll walk you through the thought process of building a searchable product data table using React.
+Isa sa mga malaking parte ng React ay kung paano ka mapapaisip tungkol sa mga apps habang binubuo mo ito. Sa dokyumentong ito, ipapakita namin sa inyo ang thought process sa pagbuo ng searchable product data table gamit ang React.
```
Comment (elrmndo) [inline content/docs/thinking-in-react.md:13]:
```suggestion
Isa sa malaking parte ng React ay kung paano ka magisip habang bumubuo ka ng apps. Sa dokyumentong ito, ipapakita namin sa inyo ang thought process sa pagbuo ng searchable product data table gamit ang React.
```

File: content/docs/thinking-in-react.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 
-## Start With A Mock {#start-with-a-mock}
+## Umpisahan gamit ang Mock {#start-with-a-mock}
```
Comment (elrmndo) [inline content/docs/thinking-in-react.md:15]:
```suggestion
## Magumpisa gamit ang isang Mock {#start-with-a-mock}
```

File: content/docs/thinking-in-react.md [hunk 4]
```diff
@@ -31,116 +31,117 @@ Our JSON API returns some data that looks like this:
 ];
 ```
 
-## Step 1: Break The UI Into A Component Hierarchy {#step-1-break-the-ui-into-a-component-hierarchy}
+## Ika-1 Hakbang: Hatiin ang UI sa isang Component Hierarchy {#step-1-break-the-ui-into-a-component-hierarchy}
 
-The first thing you'll want to do is to draw boxes around every component (and subcomponent) in the mock and give them all names. If you're working with a designer, they may have already done this, so go talk to them! Their Photoshop layer names may end up being the names of your React components!
+Ang una mong dapat gawin ay gumuhit ng mga kahon sa paligid ng bawat component (at subcomponent) sa mock at bigyan mo lahat ito ng pangalan. Kung ikaw ay may katuwang na designer, maaring sila mismo ay nagawa na ito, bilis at makipag-ugnayan sa kanila! Ang kanilang Photoshop layer names ay maaring pangalan ng iyong React components.
```
Comment (elrmndo) [inline content/docs/thinking-in-react.md:36]:
```suggestion
Ang una mong dapat gawin ay gumuhit ng mga kahon sa paligid ng bawat component (at subcomponent) sa mock at bigyan mo lahat ito ng pangalan. Kung ikaw ay may katuwang na designer, maaring nagawa na ito, at maaari kang makipag-ugnayan sa kanila. Maaari mo ding pangalanan ang iyong React components gamit ang kanilang Photoshop layer name.
```

File: content/docs/thinking-in-react.md [hunk 5]
```diff
@@ -31,116 +31,117 @@ Our JSON API returns some data that looks like this:
...
 
-But how do you know what should be its own component? Use the same techniques for deciding if you should create a new function or object. One such technique is the [single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle), that is, a component should ideally only do one thing. If it ends up growing, it should be decomposed into smaller subcomponents.
+Ngunit paano mo malalaman kung ano ba dapat ang iyong mga component? Gamiting ang parehong pamamaraan sa pagpasya kung ikaw ay gagawa ng bagong function o object. Isang pamamaraan ay ang [single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle), ito ay, kung saan ang component ay dapat isa lang ang tungkulin. Kung ang component ay lumalaki, dapat na mahati ito sa mas maliliit pang subcomponents.
```
Comment (elrmndo) [inline content/docs/thinking-in-react.md:38]:
```suggestion
Ngunit paano mo malalaman kung anong mga component ang iyong gagawin? Gamiting ang iyong pamamaraan sa pagpasya bago ka gumawa ng bagong function o object. O kaya gamitin ang [single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle) na nagsasabing ang isang component ay dapat isa lang ang tungkulin. Kung ang component ay lumalaki, dapat na mahati ito sa mas maliliit pang subcomponents.
```

File: content/docs/thinking-in-react.md [hunk 6]
```diff
@@ -31,116 +31,117 @@ Our JSON API returns some data that looks like this:
...
 
-Since you're often displaying a JSON data model to a user, you'll find that if your model was built correctly, your UI (and therefore your component structure) will map nicely. That's because UI and data models tend to adhere to the same *information architecture*. Separate your UI into components, where each component matches one piece of your data model.
+Dahil madalas kang nagpapakita ng JSON data model sa user, mapapasin mo na kung ang iyong model ay tama ang pagkakagawa, ang iyong UI (at pati ang mga component structure) ay sasangayon ng maayos. Sa kadahilanang ang UI at data models ay sumusunod sa kaparehong *information architecture*. Hatiin ang iyong UI sa mga components, kung saan ang bawat compoent ay tugma sa iyong data model.
```
Comment (elrmndo) [inline content/docs/thinking-in-react.md:40]:
```suggestion
Dahil madalas kang nagpapakita ng JSON data model sa user, mapapasin mo kung tama ang pagkakagawa ng iyong model, ang iyong UI (at pati ang iyong component structure) ay sasangayon ng maayos. Sa kadahilanang ang UI at data models ay sumusunod sa kaparehong *information architecture*. Hatiin ang iyong UI sa mga components, kung saan ang bawat component ay tugma sa iyong data model.
```

File: content/docs/thinking-in-react.md [hunk 7]
```diff
@@ -31,116 +31,117 @@ Our JSON API returns some data that looks like this:
...
 
-To build a static version of your app that renders your data model, you'll want to build components that reuse other components and pass data using *props*. *props* are a way of passing data from parent to child. If you're familiar with the concept of *state*, **don't use state at all** to build this static version. State is reserved only for interactivity, that is, data that changes over time. Since this is a static version of the app, you don't need it.
+Upang makabuo ka ng static version ng app na magrerender ng iyong data model, gugustuhin mong bumuo muna ng components na maaaring gamitin pa ng iba pang components at magpasa ng data gamit ang *props*. Ang *props* ay paraan ng pagpasa ng data mula sa parent papunta sa child component. Kung ikaw ay pamilyar sa concept ng *state*, **huwag lang gumamit ng state** upang bumuo ng static version. Ang State ay nakareserba lamang para sa interactivity, yan ay, sa mga data nagbabago sa paglipas ng panahon.
```
Comment (elrmndo) [inline content/docs/thinking-in-react.md:69]:
```suggestion
Upang makabuo ka ng static version ng app na magrerender ng iyong data model, gugustuhin mong bumuo muna ng components na maaaring gamitin pa ng iba pang components at magpasa ng data gamit ang *props*. Ang *props* ay paraan ng pagpasa ng data mula sa parent papunta sa child component. Kung ikaw ay pamilyar sa concept ng *state*, **huwag kang gumamit ng state** upang bumuo ng static version. Ang State ay nakareserba lamang para sa interactivity, yan ay, sa mga data na nagbabago o *dynamic*.
```

**Comment by warrenca** [comment]
Hi @elrmndo done with the suggestions again. Although the vercel deployment failed. Is there a manual way to trigger the deployment?

**Review by elrmndo — APPROVED** [review]

**Comment by elrmndo** [comment]
> Hi @elrmndo done with the suggestions again. Although the vercel deployment failed. Is there a manual way to trigger the deployment?

I'm not really sure why build on vercel failed since I have no directly access on it, anws I have merged this PR. Thanks! @warrenca

**Pull request merged**